sample_data/organizations.json

```diff
@@ -80,45 +80,6 @@
     "createdAt": "2023-04-13T05:16:52.827Z",
     "__v": 0
   },
-  {
-    "_id": "6637904485008f171cf29924",
-    "status": "ACTIVE",
-    "members": [
-      "66378abd85008f171cf2990d",
```

Comment:
1. Why was this removed?
2. Please restore
3. We don't want any existing functionality to be changed in importing sample data

Reply:
Revert all changes to this file.